### DISCUSSION TITLE
Is there an API that I can use for the code search technology preview?

### DISCUSSION THREAD

**Discussion by ericfitz** [community/community#36709] [Other Feature Feedback, Questions, & Ideas] [answered]
I'm using the code search technology preview, and I love it.  I am doing a search that has around 14k results in my repository, and I want to compile those results programmatically.  Is there an API I can use?

Thanks,
Eric

**Comment by ossanna16** [comment]
Hey there @ericfitz and thank you for asking a good question! I'm glad you found our community 🙂

**Comment by look** [comment, marked as answer]
@ericfitz this is on our roadmap, but there's no public API yet. Please stay tuned.
